ark/utils/spatial_analysis_utils_test.py

```diff
@@ -74,6 +75,15 @@ def make_example_data_closenum():
     dist_mat[8, 7] = 50
     dist_mat[7, 8] = 50
 
+    # add some randomization to the ordering
```

Comment:
I don't understand this part

Reply:
I should clarify this comment. Since we're not enforcing a specific ordering on distance matrices, I thought it would be nice to add a little random shuffling to the test distance matrix created to show that with the new dist_mat format, the functionality still passes.

Let me know if you think this is necessary or if we should revert back to the original.

Reply:
I was getting confused with np.ix_, assuming that was related to the coords, but now I see they're unrelated. 
Totally agree, we definitely want to check this